node/build.rs

```diff
@@ -4,7 +4,6 @@ use vergen::ConstantsFlags;
 
 fn main() {
     let mut flags = ConstantsFlags::empty();
-    flags.toggle(ConstantsFlags::SEMVER_LIGHTWEIGHT);
```

Comment:
We won't be using semver anymore?

Reply:
We will - it's just that the semver will be drawn from the `package.version` in node's Cargo.toml.